apps/frontend/src/components/Decorators/Contexable.tsx

```diff
@@ -0,0 +1,33 @@
+import { useContextMenu } from '@/hooks/useContextMenu';
+import { ContextMenuAction } from '@/types/contextMenu';
+import { MouseEvent } from 'react';
+import ContextMenu from '../ContextMenu/ContextMenu';
+
+// TODO Renames the component
+interface ContextMenuProps {
+    children: React.ReactElement;
+    actions: ContextMenuAction[];
+    className: string;
+}
+const Contextable: React.FunctionComponent<ContextMenuProps> = (props) => {
+    // destructure our state and set state functions from our custom hook
+    const { clicked, setClicked, coords, setCoords } = useContextMenu();
+    function onContextMenu(evt: MouseEvent<HTMLDivElement>) {
```

Comment:
Is not `handleContextMenu` better for the handler name ?

Reply:
Yes